backend/requirements.txt

```diff
@@ -1,9 +1,13 @@
 asgiref==3.4.1
 Django==3.2.8
 django-cors-headers==3.10.0
-django-rest-auth==0.9.5
+django-crispy-forms==1.13.0
+django-environ==0.8.1
 django-extensions==3.1.5
+django-rest-auth==0.9.5
 djangorestframework==3.12.4
 djangorestframework-simplejwt==5.0.0
+PyJWT==2.3.0
```

Comment:
Wrong requirements. I think no new requirement was added in the backend in this issue so the requirements.txt should be the same as in https://github.com/h3ssto/ddueruem-web/pull/29/files#diff-a780f027052bed3bff6ce9850db4a0e66358b86503cf4586e846ce88afa2e9bc